spynnaker8/models/synapse_dynamics/synapse_dynamics_structural.py

```diff
@@ -0,0 +1,69 @@
+from spynnaker.pyNN.models.neuron.synapse_dynamics \
+    import SynapseDynamicsStructural as _BaseClass
+from spinn_front_end_common.utilities import globals_variables
+import numpy as np
+
+from spynnaker8.models.synapse_dynamics import SynapseDynamicsStatic
+
+
+class SynapseDynamicsStructural(_BaseClass):
+    __slots__ = [
+        "_f_rew",
```

Comment:
comment the variables stating what they do/are

Reply:
most of these slots are not slots for this file, but for the SynapseDynamicsStructural. I dont think yuo need to list these, as they should be slotted in SynapseDynamicsStructural. you only ened to slot self. params of this class

Reply:
as in weights and delays